rpm/sailfishos-chum-gui.spec

```diff
@@ -92,7 +92,8 @@ Links:
 
 %build
 # SFOS RPM cmake macro disables RPATH
-%cmake -DCHUMGUI_VERSION=%{version}  \
+%cmake -DCHUMGUI_VERSION=%(echo %{version} | grep -Eo '^[0-9]+(\.[0-9]+)*')  \
```

Comment:
IMO this is superfluous, this is why I threw it out long ago: See lines 3 & 4, "The %{version} tag must adhere to semantic versioning".

If one wants arbitrary stuff in the release string, it must be put into the %{release} tag.

Is there anything I missed, i.e. why reintroducing this is necessary?

Reply:
Yes, it is necessary.

Without reverting your earlier change, OBS fails to build versions that are not tagged. Which is bad as we cannot test builds at OBS - the main target env of this app - while developing it. Hence I changed your change back.

I don't see much of a difference between your proposed regex and the one used in the commit. As I am not expert in it, I don't feel comfortable in changing it and would prefer not to invest lots of time into checking them. If you have strong opinion on it, please change it in the branch directly. But processing is needed to ensure builds at OBS for all occasions

Reply:
> If you have strong opinion on it, please change it in the branch directly.

Done and rectified some comments according to this change and my freshly obtained [knowledge about the `%sailfishos_version` string in the Sailfish-OBS](https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/285#discussion_r1613714926) by commit f4a8708.